modules/react/breadcrumbs/stories/Breadcrumbs.stories.mdx

```diff
@@ -0,0 +1,260 @@
+import {Meta} from '@storybook/addon-docs';
+import {Breadcrumbs} from '@workday/canvas-kit-react/breadcrumbs';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {Basic} from './examples/Basic';
+import {OverflowBreadcrumbs} from './examples/Overflow';
+import {RTLOverflowList} from './examples/RTL';
+import {CurrentItemTruncation} from './examples/CurrentItemTruncation';
+import {LinkTruncation} from './examples/LinkTruncation';
+
+import {
+  BreadcrumbsModelConfigComponent,
+  BreadcrumbsStateComponent,
+  BreadcrumbsEventsComponent,
+} from './BreadcrumbsModel.splitprops.tsx';
+
+<Meta title="Components/Navigation/Breadcrumbs/React" component={Breadcrumbs} />
+
+# Canvas Kit Breadcrumbs
+
+Breadcrumbs is a
+[compound component](/getting-started/for-developers/resources/compound-components/) that allows
+users to keep track and maintain awareness of their location as they navigate through pages,
+folders, files, etc.
+
+[> Workday Design Reference](https://canvas.workday.com/components/navigation/breadcrumbs)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Usage
+
+### Basic Example
+
+`Breadcrumbs` includes a container `Breadcrumbs` component and the following subcomponents which can
+be composed in a variety of ways: `Breadcrumbs.List`, `Breadcrumbs.Item`, `Breadcrumbs.CurrentItem`
+and `Breadcrumbs.Link`.
+
+Accessibility is built into `Breadcrumbs` in a way that allows you to create an inclusive experience
+with little additional configuration. As seen in the example below, the only ARIA attribute you'll
```

Comment:
I noticed in the story that there's no `aria-label` maybe it has a default?
<img width="1047" alt="Screen Shot 2022-09-06 at 11 11 36 AM" src="https://user-images.githubusercontent.com/7966550/188698238-e9b201b3-0ae9-4195-b587-3e4c3a7320c7.png">

Reply:
yes, it has default value of `aria-label` as `breadcrumbs`. I'll need update this docs. thanks for catching